library/spdm_responder_conformance_test_lib/spdm_responder_test_7_measurements.c

```diff
@@ -1831,6 +1998,11 @@ common_test_case_t m_spdm_test_group_measurements[] = {
      spdm_test_case_measurements_invalid_request,
      spdm_test_case_measurements_setup_version_any,
      spdm_test_case_common_teardown},
+    {SPDM_RESPONDER_TEST_CASE_MEASUREMENTS_SPDM_MEASUREMENT_BLOCK,
+     "spdm_test_case_measurements_success_measurements_block",
+     spdm_test_case_measurements_success_measurement_block,
+     spdm_test_case_measurements_setup_measurement_block,
```

Comment:
why not use spdm_test_case_measurements_setup_version_any?

Reply:
Thanks. I modified these questions